bin/circleci/do-perf-graphs

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
```

Comment:
Not sure it is worth the trouble, especially as I would prefer to keep the `[ $CIRCLE_NODE_INDEX -eq 0 ]` check out of `circle.yml` (and the Python script is not a good place either), and iterating over a list of `*.csv`s in Python is annoying...